### ISSUE TITLE
Setting unknown property: nystudio107\seomatic\models\MetaBundle::id

### ISSUE THREAD

**Issue by w3studio** [nystudio107/craft-seomatic#91]
I'm getting an error when I try and open an Entry in the CP or edit SEOmatic' Content SEO, as follows:
 
```
$behavior->$name = $value;
                return;
            }
        }
 
        if (method_exists($this, 'get' . $name)) {
            throw new InvalidCallException('Setting read-only property: ' . get_class($this) . '::' . $name);
        }
 
        throw new UnknownPropertyException('Setting unknown property: ' . get_class($this) . '::' . $name);
```

Sorry, I'm not sure whether its a field issue (this is a Craft 2 - 3 site – all else works with some depreciation errors to go through, any ideas? thanks

**Comment by khalwat** [comment]
What version of SEOmatic are you using? This is something that should have been addressed in SEOmatic 3.0.3.

**Comment by khalwat** [comment]
Addressed further in https://github.com/nystudio107/craft-seomatic/commit/66355b8c1dee69726bf28e27ba3116f9de02636e
